bitnami/kafka/templates/broker/svc-external-access.yaml

```
@@ -31,6 +31,9 @@ metadata:
 spec:
   type: {{ $.Values.externalAccess.broker.service.type }}
   {{- if eq $.Values.externalAccess.broker.service.type "LoadBalancer" }}
+  {{- if (not (empty .Values.externalAccess.broker.service.loadBalancerClass)) }}
+  loadBalancerClass: {{ .Values.externalAccess.broker.service.loadBalancerClass }}
```

A small change is required here:
```suggestion
  {{- if (not (empty $.Values.externalAccess.broker.service.loadBalancerClass)) }}
  loadBalancerClass: {{ $.Values.externalAccess.broker.service.loadBalancerClass }}
```